iothub/device/src/Transport/DeviceClientEndpointIdentitySASMux.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices.Client.Transport
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+    using System.IO;
+    using Microsoft.Azure.Devices.Shared;
+
+    /// <summary>
+    /// Represents DeviceClientEndpointIdentity which uses SAS authentication and multiplexing
+    /// </summary>
+    internal class DeviceClientEndpointIdentitySASMux : DeviceClientEndpointIdentity
```

Comment:
>SAS [](start = 47, length = 3)

nit: Sas, not SAS. To be consistent with other uses of Sas within class names

Reply:
I am going to rename it in the next check-in

---
In reply to: [252005014](https://github.com/Azure/azure-iot-sdk-csharp/pull/770#discussion_r252005014) [](ancestors = 252005014)